docs/development_environment.md

```diff
@@ -74,11 +78,23 @@ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-L
 
 From Windows Store install Ubuntu.
 
+#### For those already running Ubuntu 18.04 on WSL
+
+If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest FFmpeg version.
+
+```shell
+$ sudo add-apt-repository ppa:savoury1/ffmpeg4
+$ sudo add-apt-repository ppa:savoury1/graphics
+$ sudo add-apt-repository ppa:savoury1/multimedia
+$ sudo apt-get update
+$ sudo apt-get install ffmpeg
+```
+
```

Comment:
These are the changes from #553 let's leave those in that PR and avoid any merge conflicts

```suggestion
#### For those already running Ubuntu 18.04 on WSL

If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest FFmpeg version.

```shell
$ sudo add-apt-repository ppa:savoury1/ffmpeg4
$ sudo add-apt-repository ppa:savoury1/graphics
$ sudo add-apt-repository ppa:savoury1/multimedia
$ sudo apt-get update
$ sudo apt-get install ffmpeg
```

```